atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -134,10 +134,8 @@ def get_output(self, worker_metadatas):
                     # building for many, we know Pulp has schema 2
                     # support.
                     if len(worker_metadatas) == 1 and has_pulp_pull:
-                        exit_results = self.workflow.exit_results
-                        image_id, _ = exit_results[PLUGIN_PULP_PULL_KEY]
-                        if image_id is not None:
-                            instance['extra']['docker']['id'] = image_id
+                        if self.workflow.builder.image_id is not None:
```

Comment:
This change should be moved to a new PR

Reply:
agreed with Tim & Lui that it might be in this PR as 2nd commit